app/components/text_block_with_one_image_component.rb

```diff
@@ -0,0 +1,16 @@
+class TextBlockWithOneImageComponent < ViewComponent::Base
+  attr_reader :heading, :colour
+
+  def initialize(heading:, image_path:, image_alt:, colour: "pink")
+    super
+
+    @heading = heading
+    @colour = colour
+    @image_path = image_path
```

Comment:
I expect `image_path` and `image_alt` are for internal to the component class and therefore don't require getters ?

Reply:
Yeah it looks like they're only used in the `image` method below so don't need getters (for some reason GitHub is showing this as a new file in the diff but its actually all existing code and I just moved it)